frontend/src/data/demo/ui.ts

```diff
@@ -33,16 +33,19 @@ export const dashboardFilters = [
      {
 	 name: 'Gender',
 	 id: 'gender',
+     alias: dims.Dimension2Text,
```

Comment:
we're copying this object to each filter. another way we could do this with less copying is to turn `options`, below, into an array of maps- something like `{id: <item from genderDomain>, name: <dims.Dimension2Text[<item from genderDomain>>]}`; then, in [filterchip](https://github.com/Spelman-College/spelman-dashboard/pull/172/files#diff-508a6096a46fcc61c5fc230ecf9d835a5b4d8bd5ab497fb4f875fd1654a6dba3R139), display `option.name` and set `:value=option.id`.

Reply:
cancel this- adding to a map won't copy the original map.